spec/jobs/update_dqt_trn_request_job_spec.rb

```diff
@@ -6,24 +6,130 @@
   describe "#perform" do
     subject(:perform) { described_class.new.perform(dqt_trn_request) }
 
-    let(:dqt_trn_request) { create(:dqt_trn_request) }
+    let(:perform_rescue_exception) do
+      perform
+    rescue StandardError # rubocop:disable Lint/SuppressedException
```

Comment:
Shall we keep all of these in the `.rubocop.yml` so they're not littered about the place?